### DISCUSSION TITLE
Error: P1001: Can't reach database server at `******.eu-west-2.psdb.cloud`

### DISCUSSION THREAD

**Discussion by abriginets** [planetscale/discussion#82] [Bug Reports] [answered]
Hello, I'm trying out Prisma with PlanetScale it it's not going so smooth because I'm getting the following error after typing `npm prisma migrate dev`:
```
Error: P1001: Can't reach database server at `g8oms8wlmppb.eu-west-2.psdb.cloud`:`3306`

Please make sure your database server is running at `g8oms8wlmppb.eu-west-2.psdb.cloud`:`3306`.
```

What I did in the first place:
1. Created database via app.planetscale.com
2. Created new branch out of `main`
3. Copied both `main` and `dev` branch URLs to Vercel
4. Executed `vercel pull` to get my env variables (so now prod URL points to `main` and shadow database URL points to `dev` branch)
5. Created my first schema at `schema.prisma`
6. Executed `npm prisma migrate dev`

I ran a portscan against my database host and it looks like port 3306 is open and listening to connections. I tried regenerating the password and updated the URL and it didn't help so not sure what went wrong.

UPD: I'm able to connect to database using MySQL WOrkbench
UPD 2: the unsuccessfull attemp to connect happening in WSL2 environment.

**Comment by jgreet** [comment, marked as answer]
Since the server exists and is reachable, it seems possible this an SSL configuration error.  Could you verify that your database URL contains the parameters we show for Prisma connection strings:

`sslmode=require&sslaccept=strict&sslcert=/path/to/root-certs`

And that the `sslcert=` path is valid for the Linux system you've installed with WSL?

If you're using Ubuntu that path should be `/etc/ssl/certs/ca-certificates.crt` and paths for many other distributions can be found [in our docs](https://docs.planetscale.com/reference/secure-connections#ca-root-configuration).

**Reply by janpio** [reply]
Yep, that was the issue. Solved and explained over here at Prisma: https://github.com/prisma/prisma/issues/8875#issuecomment-904648630

TLDR: Make sure to use the certificate path of the environment you are running the Prisma CLI or Prisma Client in. This might be different between your local environment and where you deploy to.

(This will get easier via https://github.com/prisma/prisma/issues/8843 which will make it obsolete to supply an explicit cert path for Prisma and PlanetScale)
